benefits/oauth/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from . import views
+
+
+app_name = "oauth"
+urlpatterns = [
+    # /oauth
+    path("login", views.login, name="login"),
+    path("authorize", views.verify, name="authorize"),
```

Comment:
I'm getting an error when the devcontainer tries to run `bin/init.sh`. It's saying there is no `verify` function in `oauth/views.py`. I'm guessing that should be `views.authorize` instead?

```
  File "/home/calitp/app/benefits/oauth/urls.py", line 10, in <module>
    path("authorize", views.verify, name="authorize"),
AttributeError: module 'benefits.oauth.views' has no attribute 'verify'
```

Reply:
Fixed in [5ff71c9](https://github.com/cal-itp/benefits/pull/414/commits/5ff71c9a6a2c927f3e5dbf14fb66a942ef2b4f4f)